website/views/pages/try-fleet/log-into-fleet-sandbox-and-redirect.ejs

```diff
@@ -0,0 +1,8 @@
+<div id="log-into-fleet-sandbox-and-redirect" v-cloak>
+  <form method="post" action="<%=me.fleetSandboxURL%>/api/latest/fleet/demologin" id="demologin">
```

Comment:
I would add a comment here (inside the `<div>` otherwise it could cause Vue.js issues!!), like:

```
<!-- Confused?  Understandable, this approach is a bit unusual. See this page's view action for more info on what this code is doing and why, as well as a link where you can read more information. -->
```